recipe/meta.yaml

```diff
@@ -1,34 +1,79 @@
+# Notes: this receipt is based on torchvision-feedstock: https://github.com/conda-forge/torchvision-feedstock/blob/master/recipe/meta.yaml
+
 {% set name = "torchani" %}
-{% set version = "2.2" %}
+{% set version = "2.2.1" %}
+{% set torchani_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: c086fe6cabe49a0408d1f1b12aaf14e33b9e6ef0ffb15f3020792934e7c9fbbe
-  
+  git_url: https://github.com/aiqm/{{ name }}.git
+  git_rev: {{ version }}
+
```

Comment:
Please switch to an URL with a `sha256`. Conda forge does not like relying on git for source code since a git history can easily be overwritten. Which is not possible with a `sha256`.

When possible we prefer using the pypi archive since a bot can open PR for new version automatically:

```yaml
source:
  url: https://pypi.io/packages/source/p/pygeos/pygeos-{{ version }}.tar.gz
  sha256: 08cd7fe047a7d7f9f94eaa9a65e0b4fd4057ffebc15e6179a0290a9576a0bbbc
```

but if you need file from the git repo then you can use a GitHub archive: 

```
https://github.com/aiqm/torchani/archive/refs/tags/{{ version }}.tar.gz
```

Reply:
Historically they seem to release source. Maybe you can wait util they release?

Reply:
Hi, thanks for the quick reveiw!
I just tried and found that `archive/refs/tags/{{ version }}.tar.gz` does not download `.git` folder, which is needed for `setuptools-scm`.

errror message:
```
LookupError: setuptools-scm was unable to detect version for '/home/conda/feedstock_root/build_artifacts/torchani_1635689355009/work'
```

Reply:
do you have other suggestions?
I saw openmm is using the same way: https://github.com/conda-forge/openmm-torch-feedstock/blob/master/recipe/meta.yaml#L9

Reply:
In that case, I believe it's ok.